MakingItBetterG55/src/application/MainSceneHandler.java

```diff
@@ -1,69 +1,235 @@
 package application;
 
+import javafx.geometry.HPos;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.geometry.VPos;
 import javafx.scene.Scene;
+import javafx.scene.control.Label;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.KeyEvent;
-import javafx.scene.layout.GridPane;
-import javafx.scene.layout.Pane;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+
+import java.util.concurrent.atomic.AtomicInteger;
 
 public class MainSceneHandler {
-	
+
 	private int turnNumber = 0;
 	private boolean canMove = true;
 	private BoardHandler boardHandler;
-	
+
 	public void handleTurn() {
 		if(turnNumber > 0) {
 			boardHandler.moveToNextPlayer(turnNumber % 4);
 		}
 		switch(turnNumber % 4) {
-		case 0:
-			break;
-		case 1:
-			break;
-		case 2:
-			break;
-		case 3:
-			break;
+			case 0:
```

Comment:
Not that it matters, but no indentation before case is probably the more standard formatting for switch statements and aligned with Oracle style standards, This is probably you using the auto formatter on intellij which I believes defaults to your way whereas on eclipse it does mine, aka Eclipse > Intellij